starter-code/css/main.css

```diff
@@ -0,0 +1,75 @@
+#cover{
+    background: url("../img/slider-1.jpg") no-repeat center center;
+    background-size:cover;
+    padding:250px;
+}
+.navbar{
+    padding:24px;
+    text-align: center;
+}
+.navbar li{
+    display:inline-block;
+    margin-left: 10px;
+    
+}
+.navbar ul{
+    margin-left: 100px;
+    list-style: none;
+    text-decoration: none;
```

Comment:
Here "text-decoration" doesn't touch the text in li-s. Actually you have to do something like:
```
.navbar ul li a{
  text-decoration: none;
} 
```